endoreg_db/views/media/pdf_media.py

```
@@ -0,0 +1,386 @@
+            if not pdf.file or not pdf.file.name:
+                raise Http404("PDF file not found")
+                
+            try:
+                # Open file for streaming
+                file_path = pdf.file.path
+                if not os.path.exists(file_path):
+                    raise Http404("PDF file does not exist on disk")
+                    
+                # Create file response
+                response = FileResponse(
+                    open(file_path, 'rb'),
```

Pylint (4.0.0)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)